src/helpers.js

```diff
@@ -570,6 +570,9 @@ export function solveUpdate(affectedPaths, comparedPaths) {
  * @return {array}                 - The spliced array.
  */
 export function splice(array, startIndex, nb, ...elements) {
+  if (nb === undefined) {
```

Comment:
With Array.prototype.splice, what occurs if we do something like `arr.splice(2, null)`? Maybe you should rely on `arguments.length` or check if `nb` is a fasly value but not 0?